styles/Button.module.scss

```diff
@@ -0,0 +1,20 @@
+.socialBtn {
+  @apply flex gap-4;
+  @apply justify-center;
+  @apply bg-socials;
+  @apply rounded-md;
+  @apply w-10 h-10;
+  @apply p-1;
+  @apply shadow-md;
+  @apply hover:shadow-lg;
+}
+
+.lightBtn {
+  @apply bg-color-2;
+  @apply rounded-md border-2 border-black;
+  @apply p-2;
+  @apply font-bold;
+}
+
+.navbarBtn {
```

Comment:
comment this out for now if unused.

add the `active` style stuff here.

Reply:
Fixed